packages/markup-include/src/_stories/MarkupInclude.stories.tsx

```diff
@@ -1,18 +1,316 @@
+        borderColorTheme: false
+      }
+    },
+    isEditor: false,
+    showEditorPanel: true
+  }
+}
+
+export const With_conditions: Story = {
+  args: {
+    config: {
+      contentEditor: {
+        inlineHTML: '<div>{{state}} does have a rate 130 compared to the over all rate of {{overall-rate}}</div>',
+        markupVariables: [
+          {
+            name: 'state',
+            tag: '{{state}}',
+            columnName: 'STATE',
+            conditions: [{ columnName: 'Rate', isOrIsNotEqualTo: 'is', value: '130' }]
+          },
+          {
+            name: 'overall-rate',
+            tag: '{{overall-rate}}',
+            columnName: 'Rate',
+            conditions: [{ columnName: 'STATE', isOrIsNotEqualTo: 'is', value: 'Overall' }]
+          }
+        ],
+        showHeader: true,
+        srcUrl: '#example',
+        title: '',
+        useInlineHTML: true
+      },
+      data: [
+        {
+          STATE: 'Overall',
+          Rate: '80',
+          Location: 'Vehicle',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Alabama',
+          Rate: '130',
+          Location: 'Vehicle',
+          URL: 'https://www.cdc.gov/'
+        }
+      ],
+      legend: {},
+      newViz: true,
+      theme: 'theme-amber',
+      type: 'markup-include',
+      runtime: null,
+      visual: {
+        border: false,
+        accent: true,
+        background: true,
+        hideBackgroundColor: false,
+        borderColorTheme: false
+      }
+    },
+    isEditor: false,
+    showEditorPanel: true
+  }
+}
+
+export const Button_and_text: Story = {
+  args: {
+    config: {
+      contentEditor: {
+        inlineHTML: '<div>{{state}} does have a rate 130 compared to the over all rate of {{overall-rate}}</div>',
+        markupVariables: [],
+        showHeader: false,
+        srcUrl: 'https://www.cdc.gov/wcms/4.0/cdc-wp/data-presentation/examples/Markup-Include-Button-and-Text.html',
+        title: '',
+        useInlineHTML: true
+      },
+      data: [
+        {
+          Race: 'Hispanic or Latino',
+          'Age-adjusted rate': '644.2'
+        },
+        {
+          Race: 'Non-Hispanic American Indian',
+          'Age-adjusted rate': '636.1'
+        },
+        {
+          Race: 'Non-Hispanic Black',
+          'Age-adjusted rate': '563.7'
+        },
+        {
+          Race: 'Non-Hispanic Asian or Pacific Islander',
+          'Age-adjusted rate': '202.5'
+        },
+        {
+          Race: 'Non-Hispanic White',
+          'Age-adjusted rate': '183.6'
+        }
+      ],
+      legend: {},
+      newViz: true,
+      theme: 'theme-amber',
+      type: 'markup-include',
+      runtime: null,
+      visual: {
+        border: false,
+        accent: true,
+        background: true,
+        hideBackgroundColor: false,
+        borderColorTheme: false
+      }
+    },
+    isEditor: true,
+    showEditorPanel: true
+  }
+}
+
+export const icon_no_text: Story = {

```

Comment:
All these examples that were migrated should be imported and used directly as config